doc/source/components/services.txt

```diff
@@ -32,7 +32,7 @@ Alternatively to using Supervisor you can register each program as a system-wide
     class Tick(Component):
 
         def configure(self):
-            self += File('tick.sh', mode=0755)
+            self += File('tick.sh', mode=0o755)
```

Comment:
This could also be "rwxr-xr-x" today.

Reply:
"rwxr-xr-x" as a comment? Or 0o755, I'd rather the reader be aware, that both work